torchvision/transforms/transforms.py

```diff
@@ -533,13 +533,18 @@ class RandomPerspective(object):
     """Performs Perspective transformation of the given PIL Image randomly with a given probability.
 
     Args:
-        p (float): probability of the image being perspectively transformed. Default value is 0.5
         interpolation : Default- Image.BICUBIC
+        
+        p (float): probability of the image being perspectively transformed. Default value is 0.5
+        
+        distortion_scale(float): it controls the degree of distortion and ranges from 0 to 1. Default value is 0.5.
+    
     """
 
-    def __init__(self, interpolation=Image.BICUBIC, p=0.5):
+    def __init__(self, interpolation=Image.BICUBIC, p=0.5, distortion_scale=0.5):
```

Comment:
now that I look at this, the order of the arguments is weird, I'd think it makes more sense to let `distortion_scale` be the first one, and `interpolation` be the last

Reply:
Cool will make the changes and push !!

Reply:
@fmassa made the necessary changes !!